pkg/webui/console/views/overview/index.js

```diff
@@ -247,6 +249,23 @@ export default class Overview extends React.Component {
                   component="h2"
                 />
               )}
+              <Message
+                className={style.getStarted}
+                content={supportLink ? m.needHelp : m.needHelpWithoutSupport}
```

Comment:
What if there is no link to docs?
We do check for it in [footer](https://github.com/TheThingsNetwork/lorawan-stack/blob/issue/2770-add-documentation-link/pkg/webui/components/footer/index.js#L53-L57). IMO, we should check for it here as well.